source/extensions/http/header_validators/envoy_default/http2_header_validator.h

```diff
@@ -52,6 +77,17 @@ class Http2HeaderValidator : public HeaderValidator {
   HeaderEntryValidationResult validateResponseHeaderEntry(const ::Envoy::Http::HeaderString& key,
                                                           const ::Envoy::Http::HeaderString& value);
 
+  // Relax validation of character set in the :path header by allowing extended ASCII
```

Comment:
It looks like this method tests if `value` is valid or not (based on the characters it contains). But the comment starts with "Relax validation" which reads slightly like it's taking the action of changing validation of something. Mind rephrasing?

Reply:
Done